program/tests/helpers/mod.rs

```diff
@@ -43,8 +42,8 @@ pub fn program_test(enable_minimum_delegation: bool) -> ProgramTest {
 
     program_test.add_upgradeable_program_to_genesis("solana_stake_program", &stake_program::id());
     program_test.add_program("mpl_token_metadata", inline_mpl_token_metadata::id(), None);
-    program_test.add_program("spl_single_pool", id(), processor!(Processor::process));
-    program_test.prefer_bpf(false);
+    program_test.add_program("spl_single_pool", id(), None);
+    program_test.prefer_bpf(true);
```

Comment:
nit: `prefer_bpf` should default to true with https://github.com/anza-xyz/agave/blob/d3691c7bc8994edc4929b40e73ae3ff7d6ff7320/program-test/src/lib.rs#L513, so we shouldn't need this line at all